src/components/TodoList.js

```diff
@@ -0,0 +1,21 @@
+import TodoListItem from "./TodoListItem";
+import { useState } from "react";
+import styles from "./TodoListItem.module.css";
+import PropTypes from "prop-types";
+
+const TodoList = ({ todoList, onRemoveTodo }) => {
+  return (
+    <ul className={styles.TodoListContent}>
+      {todoList.map((todo) => (
+        <TodoListItem key={todo.id} task={todo} onRemoveTodo={onRemoveTodo} />
+      ))}
+    </ul>
+  );
+};
+
+TodoList.propTypes = {
+  todoList: PropTypes.array.isRequired,
```

Comment:
Stretch Goal:
What is `todoList` an array of? An array of strings, numbers, or an object (shape) that has structure!? When it comes to arrays and objects, the more specific you are in their structure, the better type checking you'll have during run-time!

<details>
<summary>Click here for answer</summary>

```js
TodoList.propTypes = {
  todoList: PropTypes.arrayOf(
    PropTypes.shape({
      id: PropTypes.string.isRequired,
      title: PropTypes.string.isRequired,
    }),
  ),
  onRemoveTodo: PropTypes.func.isRequired,
}
```
</details>

Reply:
Thank you Matt!